README.md

```diff
@@ -32,3 +32,9 @@ Note: will need to pass query params: {q: <search term>}
 Note: Search term is not case sensitive and will search within all plant types
 
 ##Post Endpoints:
+__Add Plant to Garden__
```

Comment:
FE is going to love us for adding this info!  We also need to go back before we're totally done and actually create a true Readme